### PR TITLE
Update to CoreAPI 3.0.19

### PR DESCRIPTION
**Pull request by HaydnDias**
Updates SDK to use latest CoreAPI spec 3.0.19

### CODE REVIEW

File: Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs [hunk 1]
```diff
@@ -147,7 +147,7 @@ public class TransitionsRequestBuilderGetQueryParameters
             /// <summary>Number of bank transfer transitions to retrieve.</summary>
             [QueryParameter("count")]
             public int? Count { get; set; }
-            /// <summary>Field on which to sort. Use any field in the resource model, or one of the system fields lastModifiedTime or createdTime.Prefix the field name with a hyphen (-) to sort in descending order.Omit the hyphen to sort in ascending order.</summary>
```
Comment (gilesc-cot) [inline Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs:150]: Gonna be grammar police on this bit! :P

**Review by kostia-cot — APPROVED** [review]

**Review by gilesc-cot — APPROVED** [review]

File: Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs (hunk 1 above)
Comment (kostia-cot) [reply Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs:150]: This is automatically generated from Marqeta's OpenAPI spec, so there is not much we can do to update the doc. 😅

Comment (gilesc-cot) [reply Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs:150]: Nooo! They just want to watch the world burn.. xD

Comment (gilesc-cot) [reply Marqeta.Core.Sdk/Generated/Banktransfers/Ach/Transitions/TransitionsRequestBuilder.cs:150]: Ok fair enough!

**Pull request merged**